scraper.rb

```diff
@@ -53,7 +53,8 @@ def prefix
   end
 
   def name
-    partitioned.last
+    # .sub called for specific case where Dr. is not separated from name by space
+    partitioned.last.sub('Dr.','')
```

Comment:
I think since this is a special case we can be very explicit about when we want this to apply. Perhaps it would be best to actually explicitly include his name:

```ruby
partitioned.last.sub('Dr.Bharrat', 'Bharrat')
```

Then we don't run the risk of this breaking for other people in the future.